test/parallel/test-repl-deprecations.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const repl = require('repl');
+
+test();
```

Comment:
Only the assumption that there may be future deprecations tests, and the `test()` function would run them all. I have no particular affinity for this approach though.